doc/api.md

```diff
@@ -27,332 +31,406 @@ question.
 
-  - Will refuse to unpublish a lone draft unless `allow_draft` is `true`.
-  - If `allow_draft` is `true`, will refuse to unpublish anything other than a draft.
-  - Will refuse to unpublish a redrafted document unless `discard_drafts` is `true`.
-  - Retrieves the live content item with the matching content_id and locale and changes its state to `unpublished`.
-  - Creates an `Unpublishing` with the provided details.
-  - Will update the `Unpublishing` if the document is already unpublished.
-  - Sends the gone/redirect/withdrawal to the live content store.
-  - If `vanish` then fully deletes the item from the live content store.
-  - Does not send to the draft content store (unless a draft was discarded).
-  - Does not send to the message queue.
-  - Returns 200 along with the content_id of the unpublished item.
-
-### Required request params:
-  - `content_id` the primary identifier for the content to publish.
-  - `type` the type of unpublishing to create/perform.
-    - Valid types are:
-      - "withdrawal"
-      - "redirect"
-      - "gone"
-      - "vanish"
-
-### Optional request params:
-  - `explanation`
-    - Type specific behaviour:
-      - "gone"
-        - Optional, message to display on page (TODO: Clarify this).
-      - "withdrawal"
-        - Required.
-      - "redirect"
-        - Ignored (TODO: Should this be rejected?).
-  - `alternative_path`
-    - If specified, this should be a [`base_path`](model.md#base_path).
-    - Type specific behaviour:
-      - "gone"
-        - Optional, `base_path` to turn into a URL to display on page for
-          `gone` (TODO: Express this better, and what happens if its not
-          specified)
-      - "redirect"
-        - Required, `base_path` to redirect to
-      - "withdrawal"
-        - Ignored (TODO: Should this be rejected?).
-  - `discard_drafts`
-    - Specify `true` to enable discarding a draft (if present).
-      - Anything other than `true` is considered `false`, including being
-        absent (TODO: Should anything other than `true` be considered
-        invalid?).
-  - `allow_draft`
-    - Specify that the intended item to unpublish is in the draft state, and
-      enable unpublishing it.
-      - (TODO: Be clear about the value to give?).
+Transitions a content item into an unpublished state. The content item will be
+updated or removed from the live content store depending on the unpublishing
+type. Uses [optimistic-locking](#optimistic-locking-previous_version).
+
+### Path parameters
+- [`content_id`](model.md#content_id)
+  - Identifies the content item to unpublish.
+
+### JSON attributes
+- `allow_draft` *(optional)*
+  - Boolean value, cannot be `true` if `discard_drafts` is also true.
+  - Specifies that if a draft content item is present it will be transitioned to
+  "unpublished" rather than the published version of this content item, which
+  itself will transition to "superseded".
```

Comment:
I don't think this is right. I think the important thing to get across here is that passing `allow_draft` as `true` means that it will only unpublish drafts.

I'm also not sure what is meant by the bit about published and superseded.